homeassistant/components/zha/select.py

```diff
@@ -93,12 +99,13 @@ def current_option(self) -> str | None:
         option = self._cluster_handler.data_cache.get(self._attribute_name)
         if option is None:
             return None
-        return option.name.replace("_", " ")
+        # needs fixing: Change all enums to uncapitalized snake_case
+        return option.name.lower()
 
     async def async_select_option(self, option: str) -> None:
         """Change the selected option."""
         self._cluster_handler.data_cache[self._attribute_name] = self._enum[
-            option.replace(" ", "_")
+            self._translation_keys[option]
         ]
         self.async_write_ha_state()
```

Comment:
This might be considered a breaking change, since we're now changing HA state from containing a space to not containing a space.
We kind of want/need the `_` in state though.